src/AssetManager.php

```diff
@@ -330,19 +304,19 @@ private function convertJs(AssetBundle $bundle): AssetBundle
      *
      * {@see registerJsFile()} for more details on javascript position.
      *
-     * @return AssetBundle the registered asset bundle instance
-     * @throws InvalidConfigException
+     * @throws InvalidConfigException|RuntimeException if the asset bundle does not exist or a circular dependency
+     * is detected.
      *
-     * @throws \RuntimeException if the asset bundle does not exist or a circular dependency is detected
+     * @return AssetBundle the registered asset bundle instance.
      */
-    private function registerAssetBundle(string $name, ?int $position = null): AssetBundle
+    private function registerAssetBundle(string $name, int $position = null): AssetBundle
     {
         if (!isset($this->assetBundles[$name])) {
             $bundle = $this->getBundle($name);
 
             $this->assetBundles[$name] = false;
 
-            // register dependencies
+            /** register dependencies */
```

Comment:
```suggestion
            // register dependencies
```
It's not phpdoc, just inline comment.

Reply:
Fixed.